### PR TITLE
Fix/tooltip

### PR DESCRIPTION
**Pull request by Yhuley**
added: 
- background colour
- text and link colour
- sizes
- positioning documentation

### CODE REVIEW

File: src/components/tooltip/README.md [hunk 1]
```diff
@@ -25,13 +25,19 @@ This component inherits the attributes of the **div** element and extends the fu
...
 - ref - ref;
-
-| propName | propType        | defaultValue | isRequired |
-| -------- | --------------- | ------------ | ---------- |
-| children | React.ReactNode | \_           | +          |
-| ref      | React.RefObject | -            | -          |
-| content  | React.ReactNode | -            | +          |
-| delay    | number          | 400          | \_         |
-| position | string          | top          | \_         |
+- backgoundColor - background color of the tooltip;
```
Comment (lazzystep) [inline src/components/tooltip/README.md:30]: typo: `backgoundColor -> backgroundColor`

File: src/components/tooltip/README.md [hunk 2]
```diff
@@ -25,13 +25,19 @@ This component inherits the attributes of the **div** element and extends the fu
...
+- width - width of the tooltip;
+
+| propName       | propType         | defaultValue | isRequired |
+| --------       | ---------------  | ------------ | ---------- |
+| children       | React.ReactNode  | \_           | +          |
+| ref            | React.RefObject  | -            | -          |
+| content        | React.ReactNode  | -            | +          |
+| delay          | number           | 400          | \_         |
+| position       | string           | top          | \_         |
+| backgoundColor | string           | -            | -          |
```
Comment (lazzystep) [inline src/components/tooltip/README.md:41]: same typo

File: src/components/tooltip/index.tsx [hunk 3]
```diff
@@ -17,6 +17,11 @@ export interface TooltipProps {
   offset?: [number, number];
   position?: PopperJS.Placement;
   popperProps?: React.HTMLAttributes<HTMLDivElement>;
+  backgoundColor?: string;
```
Comment (lazzystep) [inline src/components/tooltip/index.tsx:20]: same

**Comment by lazzystep** [comment]
@Yhuley I recommend you to install Code Spell Checker to your vscode. It will lint typo's and prevent you from this situations in future

File: src/components/tooltip/styles.ts [hunk 4]
```diff
@@ -96,7 +106,7 @@ export const Arrow = styled.div<{
 
     &::before {
       border-width: 10px 4px 10px 10px;
-      border-color: ${(props: any): string => ` transparent  transparent transparent ${props.mode === ThemeMode.dark ? props.theme.colors.base100 : props.theme.colors.base900}`};
+      border-color: transparent transparent transparent ${(props: any): string => {if (props.backgoundColor) return props.backgoundColor; return props.mode === ThemeMode.dark ? props.theme.colors.base100 : props.theme.colors.base900}};
```
Comment (lazzystep) [inline src/components/tooltip/styles.ts:109]: same typo

File: src/components/tooltip/styles.ts [hunk 5]
```diff
@@ -84,7 +94,7 @@ export const Arrow = styled.div<{
 
     &::before {
       border-width: 10px 10px 10px 6px;
-      border-color: ${(props: any): string => ` transparent ${props.mode === ThemeMode.dark ? props.theme.colors.base100 : props.theme.colors.base900}  transparent transparent`};
+      border-color: transparent ${(props: any): string => {if (props.backgoundColor) return props.backgoundColor; return props.mode === ThemeMode.dark ? props.theme.colors.base100 : props.theme.colors.base900}} transparent transparent;
```
Comment (lazzystep) [inline src/components/tooltip/styles.ts:97]: same typo

File: src/components/tooltip/styles.ts [hunk 6]
```diff
@@ -34,45 +37,52 @@ export const PopperBox = styled.div<{
...
   &[data-placement*='top'] {
-    bottom: 0;
+    bottom: 1px;
     left: 0;
     margin-bottom: -0.8em;
 
     &::before {
       border-width: 10px 10px 4px 10px;
-      border-color: ${(props: any): string => ` ${props.mode === ThemeMode.dark ? props.theme.colors.base100 : props.theme.colors.base900} transparent transparent transparent`};
+      border-color: ${(props: any): string => {if (props.backgoundColor) return props.backgoundColor; return props.mode === ThemeMode.dark ? props.theme.colors.base100 : props.theme.colors.base900}} transparent transparent transparent;
```
Comment (lazzystep) [inline src/components/tooltip/styles.ts:85]: same typo

**Review by lazzystep — CHANGES_REQUESTED** [review]
fix typos

File: src/components/tooltip/styles.ts (hunk 6 above)
Comment (Yhuley) [reply src/components/tooltip/styles.ts]: this is ctrl+c ctrl+v😅, sorry

**Review by GolotaMishka — APPROVED** [review]

**Review by sergdeev — APPROVED** [review]

**Pull request merged**